frontend/e2e-test/test/e2e/specs/5_employee/unit-editor.spec.ts

```diff
@@ -0,0 +1,41 @@
+// SPDX-FileCopyrightText: 2017-2020 City of Espoo
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+import { UnitDetailsPage } from '../../pages/admin/unit-details-page'
+import {
+  initializeAreaAndPersonData,
+  AreaAndPersonFixtures
+} from '../../dev-api/data-init'
+import { logConsoleMessages } from '../../utils/fixture'
+import { seppoAdminRole } from '../../config/users'
+import AdminHome from '../../pages/home'
+import { Daycare } from '../../dev-api/types'
+
+const adminHome = new AdminHome()
+const unitDetailsPage = new UnitDetailsPage()
+
+let fixtures: AreaAndPersonFixtures
+let cleanUp: () => Promise<void>
+let daycare1: Daycare
+
+fixture('Unit - unit details')
+  .meta({ type: 'regression', subType: 'units' })
+  .page(adminHome.homePage('admin'))
+  .before(async () => {
+    ;[fixtures, cleanUp] = await initializeAreaAndPersonData()
+    daycare1 = fixtures.daycareFixture
+  })
+  .beforeEach(async (t) => {
+    await t.useRole(seppoAdminRole)
+  })
+  .afterEach(logConsoleMessages)
+  .after(async () => {
+    await cleanUp()
+  })
+
+test('Admin can edit unit details', async () => {
+  await unitDetailsPage.openUnitDetailsPageById(daycare1.id)
+  await unitDetailsPage.enableUnitEditor()
+  await unitDetailsPage.fillManagerDataAndSubmitForm()
```

Comment:
Here you could also do something like
```
await unitDetailsPage.fillManagerData()
await unitDetailsPage.submitForm()
// todo: verify that the updated manager data is visible somewhere
```

Reply:
Thanks. Agreed, refactored accordingly in f2047b6